iceoryx_utils/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ set(IOX_VERSION_STRING "0.99.6")
 project(iceoryx_utils VERSION ${IOX_VERSION_STRING})
 #adjust_version()
 
+option(BUILD_SHARED_LIBS "Create shared libraries by default" ON)
```

Comment:
please document the new default behaviour in `installation.md`

Reply:
done in https://github.com/eclipse-iceoryx/iceoryx/pull/700